code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -175,6 +175,12 @@
 	results = list("golemmutationtoxin" = 1)
 	required_reagents = list("unstablemutationtoxin" = 1, "silver" = 1) //not too hard to get but also not just there in xenobio
 
+/datum/chemical_reaction/moth_mutation_toxin
+	name = "Moth Mutation Toxin"
+	id = "mothmutationtoxin"
+	results = list("mothmutationtoxin" = 1)
+	required_reagents = list("unstablemutationtoxin" = 1, "radium" = 1, "carbon" = 1) //carbon because life and radium because i think it glows or someshit
```

Comment:
I will deal with this if that PR is merged. Until then, shut up and stop bugging me.

Reply:
fwoosh can you not fucking read. its not a fucking persons job to prevent conflicts with PRs that arent merged yet. (unless its something really important like maps or sprites that would clash)

if either PR gets merged, the person with the PR that didnt get merged first can fix any conflicts. stop posting this shit everywhere.

Reply:
Its a config & race addition change vs a self slated 'rework of xenobiology' , maybe i am being *ott* but exaggerating the scope of my comments isn't helping given i've only mentioned it twice, eat a snickers bar or something. MRTY has already assured they'd work it out.